user_tools/pyproject.toml

```diff
@@ -54,6 +54,7 @@ dynamic=["entry-points", "version"]
 [project.scripts]
 spark_rapids_user_tools = "spark_rapids_pytools.wrapper:main"
 spark_rapids = "spark_rapids_tools.cmdli.tools_cli:main"
+spark_rapids_predict = "spark_rapids_tools.cmdli.predict_cli:main"
```

Comment:
you don't need this anymore.

Reply:
Thanks @amahussein! Addressed the review comments in new iteration.